src/molecules/Profile.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import GenericDropdown, {
+	ListLink,
+	ListButton,
+} from '../components/GenericDropdown';
+import styled from 'styled-components';
+import { useSignedIn } from 'hooks/useSignedIn';
+import { Link } from 'react-router-dom';
+
+const Button = styled.div`
+	padding: 1rem;
+`;
+
+const SignInLink = styled(Link)`
+	display: inline-block;
+	text-decoration: none;
+	text-align: center;
+	margin: 1em;
+	float: right;
+	color: ${({ theme }) => theme.colors.text[2]};
+	&:hover {
+		color: ${({ theme }) => theme.colors.primary[0]};
+	}
```

Comment:
This also adds a little padding to the child icon to make it less squished
```suggestion
	&:hover {
		color: ${({ theme }) => theme.colors.primary[0]};
	}
	& i {
		padding-right: 0.5rem;
	}
```